pkg/flowaggregator/flowaggregator.go

```diff
@@ -484,12 +577,13 @@ func (fa *flowAggregator) sendFlowKeyRecord(key ipfixintermediate.FlowKey, recor
 	if err := fa.set.AddRecord(record.Record.GetOrderedElementList(), templateID); err != nil {
 		return err
 	}
+	// if there is ipfix connection issue when we enable both ipfix and clickhouse, clickhouse won't be able to do cacheset and the record status won't be changed neither. Is it expected?
```

Comment:
I don't have enough context here, can you work this out with @wsquan171 ?

we shouldn't leave an open question in the code like this

Reply:
I forget to delete this line... Thanks for point it out.